.travis.yml

```diff
@@ -35,16 +35,16 @@ language: python
 matrix:
   include:
     - python: 2.6
-      env: NUMPY=numpy==1.6 MOCK=mock
+      env: NUMPY=numpy==1.6
     - python: 2.7
-      env: MOCK=mock PANDAS=pandas
+      env: PANDAS=pandas MOCK=mock
     - python: 3.3
     - python: 3.4
     - python: 3.5
     - python: 2.7
       env: TEST_ARGS=--pep8
     - python: 2.7
-      env: BUILD_DOCS=true MOCK=mock
+      env: BUILD_DOCS=true
```

Comment:
Let me confirm if our docs are building well on Python 3.5 first (so maybe save that for another PR).  Last time I tried they were *really* close, but I'm not certain they are 100%.  In the meantime, I'll put mock back in here, too.